docs/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.11
+
+ENV POETRY_VIRTUALENVS_IN_PROJECT=false
+ENV POETRY_VIRTUALENVS_CREATE=false
+ENV POETRY_HOME="/opt/poetry"
+ENV PATH="$POETRY_HOME/bin:$PATH"
+
+RUN apt-get update && apt-get install -y curl
+RUN curl -sSL https://install.python-poetry.org | python3 -
+
+COPY poetry.lock pyproject.toml ./
+RUN poetry install
+
```

Comment:
We should provide a default CMD so the behavior on docker run ... is defined

Reply:
Maybe this is already a good CMD: `sphinx-autobuild --host 0.0.0.0 /app /app/_build`